### PR TITLE
[service.subtitles.wizdom] 1.0.1

### PR DESCRIPTION
**Pull request by CaTzil**
### Description
New subtitle addon, it works with API from http://wizdom.xyz/#/

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by romanvm** [comment]
Also addon artwork (icon and fanart) now goes into `/resources` folder. And what `logo.png` is for?

**Comment by CaTzil** [comment]
It should be under `/resources`?
It will update the PR.

**Comment by romanvm** [comment]
See: http://kodi.wiki/view/Add-on_structure
Also please see the correct layout for language files: http://kodi.wiki/view/Language_support

**Comment by CaTzil** [comment]
Thank you for your comment, i will review it.

**Comment by CaTzil** [comment]
@romanvm, done, can you review it.

**Comment by romanvm** [comment]
Note my comment about Python API version in `addon.xml`. 

Also, as I said, The comments for `service.py` are not strictly mandatory, but following them will improve your code quality.

**Comment by CaTzil** [comment]
I can`t find your comment regarding `addon.xml`.
Regarding *comments* in the code, i prefer not to add... i think that if the code requires comments to explain it, it probably a bad code :]

File: service.subtitles.wizdom/addon.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.subtitles.wizdom" name="Wizdom Subs" version="1.0.1" provider-name="Wizdom">
+  <requires>
+    <import addon="xbmc.python" version="2.14.0"/>
```
Comment (enen92) [inline service.subtitles.wizdom/addon.xml]: @CaTzil version for xbmc.python should be 2.25.0. That was the comment

Comment (CaTzil) [inline service.subtitles.wizdom/addon.xml]: Thanx

**Comment by romanvm** [comment]
@CaTzil I wasn't talking about your comments in your code. I was talking about **my** comments regarding the general quality of your Python code. They should be visible on "Files changed" tab.

**Comment by romanvm** [comment]
I don't want to push, but please confirm if you will/won't be improving code style. Again, it is not strictly mandatory, so I will merge it even if you refuse.

**Comment by CaTzil** [comment]
Hi, pls merge, and i will fixe those issues for the next release.
Thanx

**Pull request merged**